vignettes/km-cycled.Rmd

```diff
@@ -0,0 +1,203 @@
+---
+title: "Estimating million km's cycled per zone in England and Wales regions"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Reproducing cycling potential estimates in UK cities}
```

Comment:
I think... updating this line to say:

```
   %\VignetteIndexEntry{Estimating million km's cycled per zone in England and Wales regions}
```

Will fix the error.

Looking good in any case!